server/src/serializers/BlockSerializer.js

```diff
@@ -0,0 +1,21 @@
+class BlockSerializer {
+  static async getDetail(block) {
+    const allowedAttributes = [
+      "squareOne", 
+      "squareTwo", 
+      "squareThree", 
+      "squareFour", 
+      "squareFive", 
+      "squareSix", 
+      "squareSeven",
+      "squareEight", 
+      "squareNine", 
+      "puzzleId"
+    ]
+
+    let serializedBlock = {}
+  }
+
```

Comment:
this is obviously unfinished. But Im curious, because this suggests a pretty remarkably new ER diagram.

Does a Box have many squares? are they called "Block" "Square" or "Cell". And is there a hasOne relationship between Box and Block? 
Would love to see an updated ER diagram